### PR TITLE
[Iss306] update load wra data model

### PR DESCRIPTION
**Pull request by BiancaMorandi**
The updates below were made:

1. make code work with latest version of the WRA data model (test performed also with the current version in dev) https://github.com/IEA-Task-43/digital_wra_data_standard/blob/dev/schema/iea43_wra_data_model.schema.json

    1.  update field names when needed (e.g logger_main_config, logger_measurement_config, floating_lidar etc)
    2.  fixed bug not recognising definitions when ending with _id

1. updated `_filter_parent_level` to do not remove properties if None
2. updated `measurements.get_table()` to remove columns if all values are None
3. updated demo file for testing WRA data model to version v1.0
4. added tests in `test_station.py`

### CODE REVIEW

**Comment by stephenholleran** [comment]
@BiancaMorandi anything to do here with regards the new version 1.2? Testing or anything?

File: brightwind/load/station.py [hunk 1]
```diff
@@ -115,15 +115,18 @@ def _rename_to_title(list_or_dict, schema):
             prefixed_name = key + PREFIX_DICT[key]['prefix_separator'] + col
             prefixed_names[prefixed_name] = {'prefix_separator': PREFIX_DICT[key]['prefix_separator'],
                                              'title_prefix': PREFIX_DICT[key]['title_prefix']}
+    list_special_cases_no_prefix = ['sensor_config.slope', 'sensor_config.offset', 'sensor_config.sensitivity',
```
Comment (stephenholleran) [inline brightwind/load/station.py:118]: Do we need to keep the 'sensor_config' ones? Just support from v1.0 onwards.

File: brightwind/load/station.py [hunk 2]
```diff
@@ -307,6 +309,13 @@ def _raise_child(dictionary, child_to_raise):
                            'slope', 'offset', 'sensitivity',
                            'notes', 'update_at']
     },
+    'logger_measurement_config': {
+        'prefix_separator': '.',
+        'title_prefix': 'Logger ',
+        'keys_to_prefix': ['height_m', 'height_reference_id', 'serial_number',
```
Comment (stephenholleran) [inline brightwind/load/station.py:315]: Is 'height_reference' part of logger measurement config?

File: brightwind/load/station.py [hunk 3]
```diff
@@ -307,6 +309,13 @@ def _raise_child(dictionary, child_to_raise):
...
+                           'slope', 'offset', 'sensitivity',
+                           'notes', 'update_at']
```
Comment (stephenholleran) [inline brightwind/load/station.py:317]: Not sure why having 'notes' and 'update_at' adds here?

File: brightwind/load/station.py [hunk 4]
```diff
@@ -757,25 +766,25 @@ def wdir_heights(self):
...
-        :param sensor_cfgs:   Sensor cfgs properties
-        :type sensor_cfgs:    list
-        :param sensors:       Sensor properties
-        :type sensors:        list
-        :param mount_arrgmts: Mounting arrangement properties
-        :type mount_arrgmts:  list
-        :return:              The properties merged together.
-        :rtype:               list(dict)
+        :param logger_measurement_cfgs: Sensor cfgs properties
+        :type logger_measurement_cfgs:  list
```
Comment (stephenholleran) [inline brightwind/load/station.py:775]: We should probably use 'configs' instead of 'cfgs'.

File: brightwind/load/station.py [hunk 5]
```diff
@@ -814,9 +824,9 @@ def __meas_point_merge(sensor_cfgs, sensors=None, mount_arrgmts=None):
     def __get_properties(self):
         meas_props = []
         for meas_point in self._meas_data_model:
-            # col_names_raised = _raise_child(meas_point, child_to_raise='column_name')
-            # sen_cfgs = _raise_child(col_names_raised, child_to_raise='sensor_config')
-            sen_cfgs = _raise_child(meas_point, child_to_raise='sensor_config')
+            logger_meas_cfgs = _raise_child(meas_point, child_to_raise='sensor_config')
```
Comment (stephenholleran) [inline brightwind/load/station.py:827]: How does 'sensor_config' work in this line? There shouldn't be any 'sensor_config'.

File: brightwind/load/station.py (hunk 1 above)
Comment (BiancaMorandi) [reply brightwind/load/station.py]: solved

File: brightwind/load/station.py (hunk 2 above)
Comment (BiancaMorandi) [reply brightwind/load/station.py:307]: solved

File: brightwind/load/station.py (hunk 4 above)
Comment (BiancaMorandi) [reply brightwind/load/station.py:767]: renamed all to logger_measurement_configs

File: brightwind/load/station.py (hunk 5 above)
Comment (BiancaMorandi) [reply brightwind/load/station.py]: removed

**Comment by BiancaMorandi** [comment]
@stephenholleran I have addressed all your comments:

1. removed `sensor_config`
2. renamed `logger_measurement_config(s)`
3. renamed `mounting arrangements`

**Pull request merged**